### PR TITLE
Updating FrugalScore metric card

### PR DESCRIPTION
**Pull request by sashavor**
removing duplicate paragraph

### CODE REVIEW

File: metrics/frugalscore/README.md [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 
 ## Metric Description
-FrugalScore is a reference-based metric for NLG models evaluation. It is based on a distillation approach that allows to learn a fixed, low cost version of any expensive NLG metric, while retaining most of its original performance.
+FrugalScore is a reference-based metric for Natural Language Generation (NLG) model evaluation. It is based on a distillation approach that allows to learn a fixed, low cost version of any expensive NLG metric, while retaining most of its original performance.
 
 The FrugalScore models are obtained by continuing the pretraining of small models on a synthetic dataset constructed using summarization, backtranslation and denoising models. During the training, the small models learn the internal mapping of the expensive metric, including any similarity function.
-
 The FrugalScore models are obtained by continuing the pretraining of small models on a synthetic dataset constructed using summarization, backtranslation and denoising models. During the training, the small models learn the internal mapping of the expensive metric, including any similarity function.
-
```
Comment (albertvillanova) [inline metrics/frugalscore/README.md:10]: The paragraph is still duplicated.

File: metrics/frugalscore/README.md [hunk 2]
```diff
@@ -2,12 +2,10 @@
 
 
 ## Metric Description
-FrugalScore is a reference-based metric for NLG models evaluation. It is based on a distillation approach that allows to learn a fixed, low cost version of any expensive NLG metric, while retaining most of its original performance.
+FrugalScore is a reference-based metric for Natural Language Generation (NLG) model evaluation. It is based on a distillation approach that allows to learn a fixed, low cost version of any expensive NLG metric, while retaining most of its original performance.
 
 The FrugalScore models are obtained by continuing the pretraining of small models on a synthetic dataset constructed using summarization, backtranslation and denoising models. During the training, the small models learn the internal mapping of the expensive metric, including any similarity function.
-
 The FrugalScore models are obtained by continuing the pretraining of small models on a synthetic dataset constructed using summarization, backtranslation and denoising models. During the training, the small models learn the internal mapping of the expensive metric, including any similarity function.
```
Comment (albertvillanova) [inline metrics/frugalscore/README.md:8]:
```suggestion

```

File: metrics/frugalscore/README.md [hunk 3]
```diff
@@ -2,12 +2,10 @@
...
-
 ## How to use
```
Comment (albertvillanova) [inline metrics/frugalscore/README.md:9]:
```suggestion

## How to use 
```

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Thanks for the fix. Just needing to remove the repeated text completely.

File: metrics/frugalscore/README.md (hunk 1 above)
Comment (sashavor) [reply metrics/frugalscore/README.md]: removed (again)!

**Review by albertvillanova — APPROVED** [review]
Fixed!

**Pull request merged**